src/utils.js

```diff
@@ -408,10 +409,20 @@ function normalizeSourceMapAfterPostcss(map, resourceContext) {
   return newMap;
 }
 
+function readPackageJson(readFileSync) {
+  return JSON.parse(
+    (readFileSync || fs.readFileSync)(
```

Comment:
You need to check that file exists first.

Otherwise it will throw `Can’t read file` error if `package.json` was in a different place.